frame/pablo/rpc/src/lib.rs

```diff
@@ -118,4 +126,25 @@ where
 			)))
 		})
 	}
+
+	fn redeemable_assets_for_given_lp_tokens(
+		&self,
+		pool_id: SafeRpcWrapper<PoolId>,
+		lp_amount: SafeRpcWrapper<Balance>,
+		at: Option<<Block as BlockT>::Hash>,
+	) -> RpcResult<RedeemableAssets<SafeRpcWrapper<AssetId>, SafeRpcWrapper<Balance>>> {
```

Comment:
@vivekvpandya should it provide balance for both assets in pair, i see no asset id paramter here either to distinguish which asset will be released